docs/components/concepts/global-user-task-listeners.md

```diff
@@ -0,0 +1,128 @@
+
+### Validation behavior
+
+On startup, the configuration is validated against the following rules, trying to correct issues where possible instead of failing startup:
+
+- Invalid event types are removed and ignored.
+- Listeners with missing information about event types or job type are removed and ignored.
+  - This also includes listeners for which only invalid event types have been defined.
+- If a listener defines both the special `all` value and a normal event type for `event-types`, the configuration is corrected to only include `all`.
+- Listeners with invalid information about retries (non-numeric or negative values) are removed and ignored.
+
+In all the above cases, a suitable warning is reported in the orchestration cluster startup log, identifying the problem and its location in the configuration.
+
+### Example configuration (YAML)
+
+```yaml
+zeebe:
+  broker:
+    experimental:
+      engine:
+        listeners:
+          task:
+            - type: "validate-task"
+              event-types:
+                - creating
+            - type: "audit-generic"
+              event-types: all
+              retries: 5
+            - type: "notify-assignee"
+              event-types:
+                - assigning
+                - updating
+                - canceling
+              after-local: true
+```
+
+### Example configuration (environment variables)
+
+```
+ZEEBE_BROKER_EXPERIMENTAL_ENGINE_LISTENERS_TASK_0_TYPE=validate-task
+ZEEBE_BROKER_EXPERIMENTAL_ENGINE_LISTENERS_TASK_0_EVENT_TYPES_0=creating
+ZEEBE_BROKER_EXPERIMENTAL_ENGINE_LISTENERS_TASK_1_TYPE=audit-generic
+ZEEBE_BROKER_EXPERIMENTAL_ENGINE_LISTENERS_TASK_1_EVENT_TYPES_0=all
+ZEEBE_BROKER_EXPERIMENTAL_ENGINE_LISTENERS_TASK_1_RETRIES=5
+ZEEBE_BROKER_EXPERIMENTAL_ENGINE_LISTENERS_TASK_2_TYPE=notify-assignee
+ZEEBE_BROKER_EXPERIMENTAL_ENGINE_LISTENERS_TASK_2_EVENT_TYPES_0=assigning
+ZEEBE_BROKER_EXPERIMENTAL_ENGINE_LISTENERS_TASK_2_EVENT_TYPES_1=updating
+ZEEBE_BROKER_EXPERIMENTAL_ENGINE_LISTENERS_TASK_2_EVENT_TYPES_2=canceling
+ZEEBE_BROKER_EXPERIMENTAL_ENGINE_LISTENERS_TASK_2_AFTER_LOCAL=true
+```
+
+### Applying changes
+
+Configuration changes take effect after a cluster restart. Use rolling restarts to avoid downtime.
+
+After the restart, the new configuration applies to new lifecycle events for both running and new instances, without the need for a redeployment of models.
+
+## Execution and ordering
+
+For a given event on a task instance:
+
+- Global listeners run in configuration order.
+- Model-level listeners run next in the order defined in the BPMN model.
+- Global listeners marked with `after-local: true` run after model-level listeners.
+
+All listeners are blocking and execute sequentially.
+
+## Limitations
```

Comment:
🔧 Before the limitations sections, an additional features section could be added that once again mentions and links each of the accessing user task data, correcting user task data, and denying the lifecycle transition topics in the main concept page.